backends/cuda-ref/ceed-cuda-ref.h

```diff
@@ -115,19 +115,19 @@ CEED_INTERN int CeedVectorCreate_Cuda(CeedSize n, CeedVector vec);
 
 CEED_INTERN int CeedElemRestrictionCreate_Cuda(CeedMemType mem_type, CeedCopyMode copy_mode, const CeedInt *indices, CeedElemRestriction r);
 
-CEED_INTERN int CeedElemRestrictionCreateBlocked_Cuda(const CeedMemType mem_type, const CeedCopyMode copy_mode, const CeedInt *indices,
-                                                      const CeedElemRestriction res);
+CEED_INTERN int CeedElemRestrictionCreateBlocked_Cuda(const CeedMemType mem_type, CeedCopyMode copy_mode, const CeedInt *indices,
```

Comment:
Not sure I understand why `mem_type` is `const`ed, but `copy_mode` isn't. Shouldn't they both be not modified by the function?

Reply:
woops, mem_type shouldn't be consted, thanks